search/src/main/java/artic/edu/search/SearchModule.kt

```diff
@@ -39,4 +46,34 @@ abstract class SearchModule {
 
     @get:ContributesAndroidInjector
     abstract val searchAudioDetailFragment: SearchAudioDetailFragment
+
+
+    @Module
+    companion object {
+
+        @JvmStatic
+        @Provides
+        @Singleton
+        @Named(SEARCH_CLIENT_API)
+        fun provideClient(): OkHttpClient {
+            return OkHttpClient.Builder().build()
```

Comment:
any reason we don't reuse existing client? also if we're using retrofit from `ApiModule`, this is effectively ignored.

Reply:
Oh, that just slipped through from the first draft of the commit. I wanted to create a copy of the retrofit and set the dataObject url on that, but @raigex suggested this particular type of reuse.

It can be deleted safely.